docs/traffic-policy/getting-started.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Getting Started
+---
+
+# Getting Started with Traffic Policy
+
+## Prerequisites
+
+1. **ngrok Account** - Ensure you have an active [ngrok](https://dashboard.ngrok.com/) account.
+2. **ngrok Agent** - Download and install the [ngrok agent](https://ngrok.com/download) for your operating system.
+
+## Step 1: Create a Traffic Policy
+
+Create a custom traffic policy by saving the following YAML configuration as `policy.yml`:
+
+```
+on_http_request:
+  - actions:
+    - type: custom-response
+      config:
+        content: "Hello, World!"
+```
+
+This policy will respond to each HTTP request with a simple “Hello, World!” message.
+
+## Step 2: Apply Your Traffic Policy
+
+Run the agent, applying the traffic policy you saved in the previous step with the `--traffic-policy-file` flag:
+
+```
+$ ngrok http 80 --traffic-policy-file policy.yml
```

Comment:
maybe call out that since we are using a custom response, you don't need to have anything running on port 80. also, it sort of strange you need this 🤷🏼

Reply:
Agree, but /shrug